test/services/payout/uphold_service_test.rb

```diff
@@ -412,7 +412,7 @@ class UpholdServiceTest < ActiveJob::TestCase
                 kind: 'referral'
               )
 
-              assert referral_payments.size > 0
+              assert_empty referral_payments
```

Comment:
So where's the test that potential payments for referrals are created? This is what this test was going, but now it asserts that they aren't created.

Reply:
Good catch! I'll fix this